frontend/src/components/experimentRuns/ExperimentRunsListContent.js

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import {connect} from "react-redux";
+import {Link} from "react-router-dom";
+import {Button, Tag} from "antd";
+
+import AppPageHeader from "../AppPageHeader";
+import PageContent from "../PageContent";
+import PaginatedTable from "../PaginatedTable";
+import ExportButton from "../ExportButton";
+
+import api, {withToken}  from "../../utils/api"
+import {listTable, setFilter, setFilterOption, clearFilters, setSortBy} from "../../modules/experimentRuns/actions";
+import {EXPERIMENT_RUN_FILTERS, SAMPLE_FILTERS} from "../filters/descriptions";
+import getFilterProps from "../filters/getFilterProps";
+import getNFilters from "../filters/getNFilters";
+import FiltersWarning from "../filters/FiltersWarning";
+import mergedListQueryParams from "../../utils/mergedListQueryParams";
+import {withContainer} from "../../utils/withItem";
+import {actionsToButtonList} from "../../utils/templateActions";
+
+
+const getTableColumns = (containersByID, experimentTypes, instruments) => [
+  {
+    title: "ID",
+    dataIndex: "id",
+    sorter: true,
+    render: (_, experimentRun) => (experimentRun.id &&
+      <Link to={`/experiment-runs/${experimentRun.id}`}>
+        Experiment {experimentRun.id}
+      </Link>),
+  },
+  {
+    title: "Experiment Type",
+    dataIndex: "experiment_type",
+    sorter: true,
+    options: experimentTypes.items.map(x => ({ label: x.workflow, value: x.workflow })), // for getFilterProps
+    render: (_, experimentRun) =>
+      <Tag>{experimentTypes.itemsByID[experimentRun.experiment_type]?.workflow}</Tag>,
+  },
+  {
+    title: "Instrument",
+    dataIndex: "instrument",
+    sorter: true,
+    options: instruments.items.map(x => ({ label: x.name, value: x.name })), // for getFilterProps
+    render: (_, experimentRun) =>
+      <div>{instruments.itemsByID[experimentRun.instrument]?.name}</div>,
+  },
```

Comment:
Not sure we need to add this right now (its not required for infinium but will be in the future) but at some point the instrument_type may be required (some instrument have names that do not match their type)

Reply:
It's in the bonus tasks ;P